src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -157,224 +172,137 @@ public void updatePID() {
     return DistanceArray[DistanceArray.length - 1].getVelocityAndAngle();
   }
 
-  public void adjustHood(double a) {
-    KHoodController.setReference(a, CANSparkMax.ControlType.kPosition);
-    // Adjusts Hood using PID control to passed angle a
-
-  }
-
   public double getVelocityInput() {
     return DShooterRPMInput.getDouble(0.0);
   }
 
   public void windFlywheel(double rpm) {
 
     // Winds Flywheel using PID control to passed rpm
-
-    // double adjustedRPM = rpm * (Constants.kGearRatioIn / Constants.kGearRatioOut); TODO:
-    // reconsider using this
-    if (rpm == 0.0) {
-      KShooterController.setReference(0.0, CANSparkMax.ControlType.kVoltage);
-    } else {
-      KShooterController.setReference(rpm, CANSparkMax.ControlType.kVelocity);
+    if(rpm == 0){
+      flywheelPID.setReference(0, CANSparkMax.ControlType.kVoltage);
+      flywheelPID.setIAccum(0);
+    } else{
+    targetRPM = rpm;
+    flywheelPID.setReference(rpm, CANSparkMax.ControlType.kVelocity);
```

Comment:
Doesn't look like spotless ran. Make sure you build the code before pushing and creating the PR

Reply:
Will do in the future, for some reason I thought that would happen when we merged to main